nmos-test.py

```diff
@@ -36,43 +37,115 @@
 app.register_blueprint(REGISTRY_API)  # Dependency for IS0401Test
 
 CACHE_PATH = 'cache'
-SPEC_REPOS = [
-    ('is-04', 'nmos-discovery-registration'),
-    ('is-05', 'nmos-device-connection-management'),
-    ('is-06', 'nmos-network-control'),
-    ('is-07', 'nmos-event-tally')
-]
+SPECIFICATIONS = {

```

Comment:
Not a big deal, but we should probably factor these big definition structs out into their own file at some point.

Reply:
Agreed. I think we can do similar for the form components.